packages/cubejs-schema-compiler/src/compiler/transpilers/ImportExportTranspiler.ts

```diff
@@ -50,7 +50,7 @@ export class ImportExportTranspiler implements TranspilerInterface {
           }
         });
         const addExportCall = t.callExpression(t.identifier('addExport'), [t.objectExpression(<t.ObjectProperty[]>declarations)]);
-        if (path.get('declaration')) {
+        if ('declaration' in path.node && path.node.declaration) {
           path.replaceWithMultiple([
             // @todo fix without any
             (<any>path.get('declaration')).node,
```

Comment:
As I can see, `path.get('declaration')` is still used in `path.replaceWithMultiple`.
Two different nodes in `if` condition and `replaceWithMultiple` call.